### PR TITLE
Make grapher-live secure for production!

### PR DESCRIPTION
**Pull request by Herteby**
I might be wrong but I think this would be all that's needed to secure grapher-live?

### CODE REVIEW

**Comment by theodorDiaconu** [comment]
Published 1.3.0_1

**Comment by Herteby** [comment]
Nice :+1:

File: ui/Live.jsx [hunk 1]
```diff
@@ -59,13 +59,14 @@ class Live extends React.Component {
...
                                 }
-
+                                {!Meteor.isProduction &&
                                 <span className="check-user">
                                     <input type="checkbox" onChange={this.onCheckForUser.bind(this)} />
                                     <span>
                                         Bypass exposure firewall
                                     </span>
                                 </span>
+                                }}
```
Comment (theodorDiaconu) [inline ui/Live.jsx:69]: wait, isn't this giving an error ?

Comment (Herteby) [reply ui/Live.jsx:69]: Oops, I didn't actually test it, I just quickly edited on GitHub as a suggestion :sweat_smile:

**Pull request merged**